tests/std/tests/P0355R7_calendars_and_time_zones_formatting/test.cpp

```diff
@@ -46,7 +46,7 @@ template <typename CharT>
 template <typename CharT>
 struct testing_callbacks {
     _Fmt_align expected_alignment = _Fmt_align::_None;
-    basic_string_view<CharT> expected_fill;
+    basic_string_view<CharT> expected_fill{};

```

Comment:
Should we add braces to `vector<_Chrono_spec<CharT>>& expected_chrono_specs;` below for consistency?

Reply:
It's not meaningful to initialize a member reference with a prvalue.